manifests/init.pp

```diff
@@ -172,8 +172,8 @@
     manage_repo              => $manage_repo,
   }
```

Comment:
@igalic You can, (and I think should), still `contain` it.

> To contain classes that are declared with the resource-like declaration syntax, use the contain function after declaring the class
https://docs.puppet.com/puppet/4.9/lang_containment.html